Help/MAC_Hints.rst

```diff
@@ -60,3 +71,44 @@ Finally!
 
     # Install python-coverage3 by
     $ easy_install coverage
+
+pyenv
+-----
+
+Here we show how to use pyenv, which is simply a python installer
+that allows you to install multiple python versions and switch
+through them. After installation, you may also want to set up
+a virtual environment using :code:`venv` or :code:`virtualenv`.
+
+1. Install pyenv with homebrew
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+::
+
+    $ brew update
+    $ brew install pyenv
+
+2. Installing and using python with pyenv
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To install python you simply have to
+do this (you don't have to install all of them) :
+
+::
+
+    $ pyenv install 3.5.0
+    $ pyenv install 3.4.3
+    $ pyenv install 3.3.6
+    $ pyenv install 3.2.6
+    $ pyenv install 2.7.10
+    $ pyenv versions # lists the versions you have
+
+To use a particular version you simply have to do this :
+
+::
+
+    $ pyenv local 3.5.0
+
```

Comment:
probably add the list of commands link here 
https://github.com/pyenv/pyenv/blob/master/COMMANDS.md